sidebars.js

```diff
@@ -88,6 +88,7 @@ const sidebars = {
             'quickstart/ccd/monitoring'
           ]
         },
+        'quickstart/rbac',
```

Comment:
This doesn't really belong in `quickstart`. This is more concepts explained. Quickstarts should be tangible, reproducible examples.